swagger_parser/lib/src/utils/utils.dart

```diff
@@ -48,6 +48,25 @@ String ioImport(UniversalComponentClass dataClass) => dataClass.parameters
     ? "import 'dart:io';\n\n"
     : '';
 
+const dartGeneratedFileComment = r'''
+//  ____ _ _ _ ____ ____ ____ ____ ____     ___  ____ ____ ____ ____ ____
```

Comment:
Also change generated text to 
```dart
// coverage:ignore-file
// GENERATED CODE - DO NOT MODIFY BY HAND
// ignore_for_file: type=lint
```

Reply:
Edited in https://github.com/Carapacik/swagger_parser/pull/90/commits/8829f6263cf6f2bffd7526dead686398b33e5532